Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt

```diff
@@ -351,7 +361,10 @@ private fun InputSelectorButton(
             Icon(
                 icon,
                 tint = tint,
-                modifier = Modifier.padding(12.dp).preferredSize(20.dp)
+                modifier = Modifier
+                    .padding(12.dp)
+                    .preferredSize(20.dp),
+                contentDescription = null
```

Comment:
Remove the `semantics` modifier of the `IconButton` and use `description` here
```suggestion
                contentDescription = description
```